ux.symfony.com/composer.json

```diff
@@ -38,7 +38,9 @@
         "symfony/ux-dropzone": "2.x-dev",
         "symfony/ux-icons": "2.x-dev",
         "symfony/ux-lazy-image": "2.x-dev",
+        "symfony/ux-leaflet-map": "^2.19",
```

Comment:
Why not using `2.x-dev` like for other ux packages?

Reply:
Mistake.. 😅  thx